packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -101,6 +103,8 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
     [sceneContentUri, baseUrl],
   );
 
+  const isViewing = config.mode === 'Viewing';
```

Comment:
nit-pick: When comparing strings I'm a big fan of using `.toLowerCase()` to ensure that the check is not case-sensitive. 

ex: `const isViewing = config.mode.toLowerCase() === 'viewing';`

Reply:
the case shouldn't have any problem because the type of mode is `'Editing' | 'Viewing'` and when you compare it with other strings, there will be an error showing `This comparison appears to be unintentional because the types 'OperationMode | undefined' and '"viewing"' have no overlap.`